lib/stringify.js

```diff
@@ -68,6 +68,12 @@ function stringifyVar(va, format, userStringifyVar, space) {
 }
 
 function quoteIfNeeded(text) {
+  if (typeof text === 'number') {
+    text = text.toString();
+  }
+  if (Array.isArray(text)) {
```

Comment:
In which situation `text` could be an array?

Reply:
When there are multiple authors.

Reply:
> Currently, usercss doesn't support multiple authors.

Stylus accepted my UserCSS with multiple authors just fine.

Reply:
> Stylus accepted my UserCSS with multiple authors just fine.

Stylus doesn't accept all authors, the latter would overwrite the former:
https://repl.it/@eight041/usercss-meta-multi-author

Reply:
The parser may only return the last one, but the UserCSS generated by `stringify` will still have all of them and Stylus won't throw any error.

Reply:
> UserCSS generated by stringify will still have all of them

That is the feature I explained [here](https://github.com/StylishThemes/parse-usercss/pull/15#issuecomment-406854065). If that is what you need, you can remove L71-L73.

Reply:
OK I amended my previous commit to remove those lines as well.